Add warning about sensitive info on traces (#9015)
kyma-project/kyma: docs/tracing/01-01-tracing.md

The microservice architecture differs from the traditional monoliths in many aspects. From the request observability perspective, there are asynchronous boundaries among various different microservices that compose a request flow. Moreover, these microservices can have heterogeneous semantics when it comes to monitoring. A tracing solution that provides a holistic view of the request flow helps you to understand the system and take informed decisions regarding troubleshooting and performance optimization.

Tracing in Kyma uses [Jaeger](https://www.jaegertracing.io/docs/) as a backend which serves as the query mechanism for displaying information about traces.

> >**CAUTION:** Jaeger is designed for application tracing. Do not send any sensitive information on the traces, such as passwords or credit card numbers.

Reviewer: ```suggestion
>**CAUTION:** Jaeger is designed for application tracing. Traces should not carry sensitive information, such as passwords or credit card numbers.
```
